doc/api/cli.md

```diff
@@ -1575,6 +1542,10 @@ added: v0.8.0
 
 Silence deprecation warnings.
 
+### `--no-experimental-detect-module`
+
```

Comment:
```suggestion

<!-- YAML
added:
  - v21.1.0
  - v20.10.0
changes:
  - version:
    - REPLACEME
    pr-url: https://github.com/nodejs/node/pull/53619
    description: Syntax detection is enabled by default.
-->

```

Reply:
If we have this in two places then it's just another thing that can get out of sync. Do we need it repeated?

Reply:
Yes I’d say so, folks reading the CLI docs would need to have the information available.